### PR TITLE
chore: add close button on popup

### LINKED ISSUE #1484: [BUG] Add a close symbol to the card pop-up.
### Description

While clicking on read more user can't see the option to close. although he can click sideways to close, but a symbol must be there for newbies and others to close

### Screenshots

image

this is the present image.

image

this is the proposed idea

### Any additional information?

_No response_

### What browser are you seeing the problem on?

_No response_

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by k-deepak04**
## closes

Closes #1484 

## Screenshots

Screenshot 2023-08-04 180130

### CODE REVIEW

**Comment by k-deepak04** [comment]
@CBID2 Please go ahead and review it 
Speed 🥷🐢.

**Review by CBID2 — DISMISSED** [review]
LGTM

**Comment by k-deepak04** [comment]
@rupali-codes please review and merge.

**Review by rupali-codes — COMMENTED** [review]
its not looking good actually, 

can you add something like _close_ text right after the _visit button_?

like this
image

**Comment by k-deepak04** [comment]
> its not looking good actually,
> 
> can you add something like _close_ text right after the _visit button_?
> 
> like this image

okay i'll do this

**Comment by k-deepak04** [comment]
image

web

image

phone

@rupali-codes @CBID2 please check and let me know

**Comment by CBID2** [comment]
Looks pretty good to me @k-deepak04

**Comment by k-deepak04** [comment]
> Looks pretty good to me @k-deepak04

thanks :)
the previous level tag was also looking pretty good on this issue 🤣🤣

**Comment by k-deepak04** [comment]
@rupali-codes please merge this one too

**Comment by rupali-codes** [comment]
@k-deepak04 it has too much spacing, could you please it? also please reduce the font-size

image

**Comment by k-deepak04** [comment]
ok will do it now

**Comment by k-deepak04** [comment]
image

will this works?

**Comment by k-deepak04** [comment]
image

phone view

**Comment by k-deepak04** [comment]
@rupali-codes let me know

File: components/popup/popupInfo.tsx [hunk 1]
```diff
@@ -66,6 +66,16 @@ export const PopupInfo: React.FC<{
               Visit site
             </a>
           </div>
+
+          {/* Close Text */}
+          <div className="flex justify-center items-center">
+            <p
+              onClick={onClose}
+              className="cursor-pointer text-xl text-theme-primary"
```
Comment (rupali-codes) [inline components/popup/popupInfo.tsx:74]:
```suggestion
              className="cursor-pointer text-sm text-theme-primary -mt-3"
```

Comment (k-deepak04) [reply components/popup/popupInfo.tsx:74]: commiting again.

**Comment by k-deepak04** [comment]
@rupali-codes commited the changes you said :)

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**